appinventor/appengine/war/static/css/Ya.css

```diff
@@ -209,49 +244,106 @@ body {
 /*
  * ProjectList and CopyYoungAndroidProjectCommand.CopyProjectDialog
  */
+.ode-ProjectFieldSpan {
+  display: flex;
+  column-gap: 4px;
+  align-items: center;
+}
 
 .ode-ProjectNameLabel {
   font-weight: bold;
   font-size: small;
   color: #555;
   cursor: pointer;
+  vertical-align: top;
+  padding-right: 10px;
+  text-align: left;
+  flex-grow: 1;
+  min-width: 100px;
 }
 
-.ode-ProjectTable {
-  padding: 0px;
-  margin: 0px;
-  border: 1px;
-}
-
-.ode-ProjectTable td {
-  border-collapse: collapse;
+.ode-ProjectFieldLabel {
   font-size: small;
   color: #555;
+  vertical-align: top;
+  padding-right: 10px;
   text-align: left;
-  margin: 0px;
+  width: 200px;
+}
+
+.ode-ProjectTable {
+  padding: 0;
+  margin: 0;
+  border: 1px;
+  overflow: auto;
+  border-radius: 4px;
+  height: auto;
+  width: 100%;
 }
 
 .ode-ProjectRowHighlighted {
-  border-bottom: 1px solid #ddd;
-  background-color: #fff6c3;
-  border-collapse: collapse;
+  background-color: #fff6c3 !important;
```

Comment:
Do you recall why having `!important` here was necessary? We should avoid using it if at all possible.

Reply:
See my note in ProjectListItem.java. CSS rules with the same rank will be applied in the order they appear in the CSS file, so by starting with the rows having both the UnHighlight and Highlight styles, the `background-color` here is immediately override by the `background-color` in `.ode-ProjectRowUnHighlighted` below.

Reply:
> Do you recall why having `!important` here was necessary? We should avoid using it if at all possible.

I wasn't able to get the correct background color to show up reliably. More insight definitely appreciated.